From a28f3475ff4bf5e20a4386d505a222b58035cb74 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Tue, 1 May 2012 15:33:39 -0400
Subject: [PATCH 4/6] Delete gdm-password from simple-greeter/extensions

It's going to be obsoleted by the next patch.  See commit
0e0aca600da17cb952525617f79ee3cadc028a8a which should have moved it.

https://bugzilla.gnome.org/show_bug.cgi?id=675085
---
 gui/simple-greeter/extensions/password/Makefile.am |    9 +--------
 .../extensions/password/gdm-password               |   19 -------------------
 .../extensions/password/gdm-password.pam           |   19 -------------------
 3 files changed, 1 insertions(+), 46 deletions(-)
 delete mode 100644 gui/simple-greeter/extensions/password/gdm-password
 delete mode 100644 gui/simple-greeter/extensions/password/gdm-password.pam

diff --git a/gui/simple-greeter/extensions/password/Makefile.am b/gui/simple-greeter/extensions/password/Makefile.am
index e15fc82..84d4729 100644
--- a/gui/simple-greeter/extensions/password/Makefile.am
+++ b/gui/simple-greeter/extensions/password/Makefile.am
@@ -36,14 +36,7 @@ libpassword_la_SOURCES =				\
 			gdm-password-extension.h	\
 			gdm-password-extension.c
 
-$(PAM_SERVICE_NAME): $(PAM_SERVICE_NAME).pam
-	cp $(srcdir)/$(PAM_SERVICE_NAME).pam $(builddir)/$(PAM_SERVICE_NAME)
-
-pamdir = $(PAM_PREFIX)/pam.d
-pam_DATA = $(PAM_SERVICE_NAME)
-
-EXTRA_DIST = $(extension_DATA) $(PAM_SERVICE_NAME).pam
-CLEANFILES = $(PAM_SERVICE_NAME)
+EXTRA_DIST = $(extension_DATA)
 
 MAINTAINERCLEANFILES =                  \
         *~                              \
diff --git a/gui/simple-greeter/extensions/password/gdm-password b/gui/simple-greeter/extensions/password/gdm-password
deleted file mode 100644
index bac431d..0000000
--- a/gui/simple-greeter/extensions/password/gdm-password
+++ /dev/null
@@ -1,19 +0,0 @@
-# Sample PAM file for doing password authentication.
-# Distros should replace this with what makes sense for them.
-auth        required      pam_env.so
-auth        sufficient    pam_unix.so nullok try_first_pass
-auth        requisite     pam_succeed_if.so uid >= 500 quiet
-auth        required      pam_deny.so
-
-account     required      pam_unix.so
-account     sufficient    pam_localuser.so
-account     sufficient    pam_succeed_if.so uid < 500 quiet
-account     required      pam_permit.so
-
-password    requisite     pam_cracklib.so try_first_pass retry=3 type=
-password    sufficient    pam_unix.so nullok try_first_pass use_authtok
-password    required      pam_deny.so
-
-session     optional      pam_keyinit.so revoke
-session     required      pam_limits.so
-session     required      pam_unix.so
diff --git a/gui/simple-greeter/extensions/password/gdm-password.pam b/gui/simple-greeter/extensions/password/gdm-password.pam
deleted file mode 100644
index bac431d..0000000
--- a/gui/simple-greeter/extensions/password/gdm-password.pam
+++ /dev/null
@@ -1,19 +0,0 @@
-# Sample PAM file for doing password authentication.
-# Distros should replace this with what makes sense for them.
-auth        required      pam_env.so
-auth        sufficient    pam_unix.so nullok try_first_pass
-auth        requisite     pam_succeed_if.so uid >= 500 quiet
-auth        required      pam_deny.so
-
-account     required      pam_unix.so
-account     sufficient    pam_localuser.so
-account     sufficient    pam_succeed_if.so uid < 500 quiet
-account     required      pam_permit.so
-
-password    requisite     pam_cracklib.so try_first_pass retry=3 type=
-password    sufficient    pam_unix.so nullok try_first_pass use_authtok
-password    required      pam_deny.so
-
-session     optional      pam_keyinit.so revoke
-session     required      pam_limits.so
-session     required      pam_unix.so
-- 
1.7.7.6

